### ISSUE TITLE
display a number of cafes found/available on general or filtered by states page

### ISSUE THREAD

**Issue by donutranger** [petakopi/petakopi.my#122]
Hi, just a suggestion I think could be a good QOL increase. 

It would be nice if there can be a counter of cafes found somewhere in the page especially when we've filtered the page by states.

![image](https://github.com/petakopi/petakopi.my/assets/18351759/dba12c48-b2d8-45ce-baca-fb5630543584)

**Comment by amree** [comment]
Good idea and thanks for the suggestion, however, this poses a different challenge when it comes to **mobile view** due to the limited space that we have right now. Any thoughts on that?

<img width="499" alt="image" src="https://github.com/petakopi/petakopi.my/assets/424392/e0ccf8b8-68d7-48d2-88d9-05c44ecbc2fc">
